samples/vanilla/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "vanilla-sample",
+  "version": "1.0.0",
+  "description": "A sample vanilla application  is developed to work with @refinitiv-ui components",
+  "main": "index.html",
+  "scripts": {
+    "start": "npm run build && webpack serve",
+    "build": "npm run build:light && npm run build:dark && webpack build --mode production",
+    "build:light": "lessc ./src/styles/light.less ./dist/styles/light.css",
+    "build:dark": "lessc ./src/styles/dark.less ./dist/styles/dark.css",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

Comment:
Can you guys make sure that it is ok to use this license or is there more appropriate one ?

Reply:
Not sure about this. I'll delete it at first.